gauguin-app/src/main/kotlin/org/piepmeyer/gauguin/ui/grid/GridFontHolder.kt

```diff
@@ -13,12 +13,22 @@ class GridFontHolder(
     val fontValue: Typeface
 
     init {
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
             val builder = Typeface.Builder(context.assets, "font/InterVariable.ttf")
 
             fontPossibles = builder.setFontVariationSettings("'wght' 400").build()
             fontValue = builder.setFontVariationSettings("'wght' 425").build()
             fontCageText = builder.setFontVariationSettings("'wght' 575").build()
+        } else if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            // Builders can't be reused in SDKs O and P
+            val builder = Typeface.Builder(context.assets, "font/InterVariable.ttf")
```

Comment:
```suggestion
            var builder = Typeface.Builder(context.assets, "font/InterVariable.ttf")
```

Reply:
Sorry about that, I’ve pushed the fix.

Reply:
No need for a sorry, thanks a lot for the fix!